common/app/controllers/IndexControllerCommon.scala

```diff
@@ -31,8 +31,8 @@ trait IndexControllerCommon
       logGoogleBot(request)
       index(leftSide, rightSide, inferPage(request), request.isRss).map {
```

Comment:
You can also do flatMap here instead of . flatten at the end (makes no difference)